mmpose/datasets/datasets/__init__.py

```diff
@@ -8,36 +8,35 @@
 from .bottom_up import (BottomUpAicDataset, BottomUpCocoDataset,
                         BottomUpCocoWholeBodyDataset, BottomUpCrowdPoseDataset,
                         BottomUpMhpDataset)
-from .face import (Face300WDataset, FaceAFLWDataset, FaceCocoWholeBodyDataset,
```

Comment:
`FaceCocoWholeBodyDataset` is mistakenly deleted.

Reply:
OK, I will add it.